### PR TITLE
Improve Windows GTK runtime bundle creation script

### PR DESCRIPTION
**Pull request by eht16**
Update the list of dependencies, include sort.exe and grep.exe,
create a information file with all download links of included
binaries and re-structure the script for better readability.
See #560 for details.

This PR actually supersedes #560.

### CODE REVIEW

File: scripts/gtk-bundle-from-msys2.sh [hunk 1]
```diff
@@ -1,109 +1,270 @@
...
+		"-n")
+			run_pi=""
+			;;
+		"-h"|"--help")
+			echo "gtk-bundle-from-msys2.sh [-c] [-z] [-2 | -3]"
+			echo "   -c Use pacman cache. Otherwise pacman will download"
+			echo "      archive files"
+			echo "   -z Create a zip afterwards"
+			echo "   -2 Prefer gtk2"
+			echo "   -3 Prefer gtk3"
```
Comment (b4n) [inline scripts/gtk-bundle-from-msys2.sh]: doesn't explain `-n` or `-h`

Comment (b4n) [inline scripts/gtk-bundle-from-msys2.sh]: not cachedir arg

**Comment by b4n** [comment]
I can't really tell, but doesn't seem to weird

File: scripts/gtk-bundle-from-msys2.sh (hunk 1 above)
Comment (eht16) [inline scripts/gtk-bundle-from-msys2.sh]: Fixed, thanks for spotting.

**Pull request merged**